.ruby-version

```diff
@@ -1 +1 @@
-jruby-1.7.17
+jruby-1.7.25
```

Comment:
Be careful here, we don't have a .25 release RPM and so we need to ensure devs don't miss bugs by using a different JRuby to what's actually deployed (we had this issue in the past you see where some devs used diff versions and so bugs in code and tests weren't seen)